validations/src/validations.rs

```diff
@@ -24,14 +29,40 @@ pub fn transaction_inputs_sum(
 ) -> Result<u64, failure::Error> {
     let mut total_value = 0;
 
-    for input in &tx.inputs {
-        let pointed_value = pool
-            .get(&input.output_pointer())
-            .ok_or_else(|| TransactionError::OutputNotFound {
-                output: input.output_pointer(),
-            })?
-            .value();
-        total_value += pointed_value;
+    match transaction_tag(tx) {
+        // Commit exception to calculate inputs
```

Comment:
_Suggestion_: remove this kind of comment, they add almost 0 value

Reply:
:crying_cat_face: